svc-lighthouse-api/src/integrationTest/java/gov/va/vro/abddataaccess/FhirClientTest.java

```diff
@@ -1,8 +1,6 @@
 package gov.va.vro.abddataaccess;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Undo this change due to [warning](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/3721229093/jobs/6311241939#step:5:880)

Reply:
done.